amazonec2/tests/test.bal

```diff
@@ -48,13 +48,13 @@ function testCreateSecurityGroup() {
 
     time:Time time = time:currentTime();
     int currentTimeMills = time.time;
-    string currentTimeStamp = <string>(currentTimeMills / 1000);
+    string currentTimeStamp = string.convert(currentTimeMills / 1000);
     var securityGroup = amazonEC2Client->createSecurityGroup("Ballerina_test_group" + currentTimeStamp,
         "Ballerina Test Group");
     if (securityGroup is SecurityGroup) {
         io:println("Successfully created the security group: ", securityGroup);
         testGroupId = untaint securityGroup.groupId;
-        test:assertNotEquals(securityGroup.groupId, null, msg = "Failed to create the security group");
+        test:assertNotEquals(securityGroup.groupId, (), msg = "Failed to create the security group");
```

Comment:
Please create an issue to improve these tests. Would be ideal if we could do an assertEquals based on some input params instead of just checking not equals to `()`.

Reply:
Created issue https://github.com/wso2-ballerina/module-amazonec2/issues/16